scene/main/viewport.cpp

```diff
@@ -1436,16 +1428,9 @@ void Viewport::_gui_input_event(Ref<InputEvent> p_event) {
 		if (mb->is_pressed()) {
 			Size2 pos = mpos;
 			if (gui.mouse_focus_mask) {
-				//do not steal mouse focus and stuff while a focus mask exists
-				gui.mouse_focus_mask |= 1 << (mb->get_button_index() - 1); //add the button to the mask
+				// Do not steal mouse focus and stuff while a focus mask exists.
+				gui.mouse_focus_mask |= 1 << (mb->get_button_index() - 1); // Add the button to the mask.
 			} else {
-				//Matrix32 parent_xform;
-
-				/*
-				if (data.parent_canvas_item)
-					parent_xform=data.parent_canvas_item->get_global_transform();
-				*/
-
```

Comment:
Commented out for 6 years in https://github.com/godotengine/godot/commit/bdfac2806fc226169f1ed576d8d0e9e677578650